features/command.feature

```diff
@@ -1180,3 +1180,125 @@ Feature: WP-CLI Commands
       Success: unknown-parent child-command
       """
     And STDERR should be empty
+
+  Scenario: The command should fire on `after_wp_load`
+    Given a WP install
+    And a custom-cmd.php file:
+      """
+      <?php
+      /**
+       * @when before_wp_load
+       */
+      class Custom_Command_Class extends WP_CLI_Command {
+          /**
+           * @when after_wp_load
+           */
+          public function home_url() {
+             WP_CLI::success( home_url() );
```

Comment:
To make these tests more precise, could we have each method `var_dump( function_exists( 'home_url' ) )`?

With this approach, we'll know definitely whether WordPress has loaded or not.